packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -366,23 +381,37 @@ export class Markets {
     if (!(await augur.contracts.augur.isKnownUniverse_(params.universe))) {
       throw new Error('Unknown universe: ' + params.universe);
     }
+    const validLiquiditySpreads = ['10', '15', '20', '100'];
+    if (params.maxLiquiditySpread && !validLiquiditySpreads.includes(params.maxLiquiditySpread)) {
+      throw new Error('Invalid maxLiquiditySpread');
+    }
+    // Set sort defaults
+    if (typeof params.sortBy === 'undefined') {
+      params.sortBy = GetMarketsSortBy.OPEN_INTEREST;
```

Comment:
We might want default sort to be either volume or something else. Need to run this by @joeykrug and @Chwy5

Reply:
Neither --- it'll be the liquidity sort, cc @nuevoalex

Reply:
Sounds good. I've updated https://github.com/AugurProject/augur/issues/3023 to include modifying the default sort to be liquidity.